internal/cmd/workerpools/flags.go

```diff
@@ -0,0 +1,21 @@
+package workerpools
+
+import "github.com/urfave/cli/v2"
+
+var flagPoolID = &cli.StringFlag{
+	Name:     "id",
+	Usage:    "[Required] ID of the worker pool",
+	Required: true,
+}
+
+var flagPoolIdNamed = &cli.StringFlag{
```

Comment:
Well, I think using the named version everywhere would make the usage quite verbose. For that reason I prefer the to use the short version, where possible.

The case of `spacectl workerpool worker list` with its named version is a special one, because the usage would otherwise be confusing. The expected result of the command is a list of workers, so I'd expect an `--id` flag to refer to the worker as well, and not the workerpool. But as the API is designed in a way, where you can only query the workers of a specific workerpool, I've added the `--pool-id` flag.

Reply:
Fair enough - you've convinced me :smile: